pkgs/development/python-modules/pymediainfo/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchPypi, buildPythonPackage
+, libmediainfo
+, pytest, glibcLocales }:
+
+buildPythonPackage rec {
+  pname = "pymediainfo";
+  version = "2.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "160yyjyfg441h30pqfr6370hcch17j3s0x2f1l0r220xgy969mr1";
+  };
+
+  postPatch = ''
+    substituteInPlace pymediainfo/__init__.py \
+      --replace 'CDLL(library_file)' \
+                'CDLL("${libmediainfo}/lib/libmediainfo${stdenv.hostPlatform.extensions.sharedLibrary}")' \
+      --replace 'CDLL("libmediainfo.0.dylib")' \
+                'CDLL("${libmediainfo}/lib/libmediainfo.0${stdenv.hostPlatform.extensions.sharedLibrary}")' \
+      --replace 'CDLL("libmediainfo.dylib")' \
+                'CDLL("${libmediainfo}/lib/libmediainfo${stdenv.hostPlatform.extensions.sharedLibrary}")' \
+      --replace 'CDLL("libmediainfo.so.0")' \
+                'CDLL("${libmediainfo}/lib/libmediainfo${stdenv.hostPlatform.extensions.sharedLibrary}.0")'
+  '';
+
+  buildInputs = [ libmediainfo glibcLocales ];
```

Comment:
`libmediainfo` should not be necessary here because it is referenced in `postPatch`. `glibcLocales` can go into `checkInputs`.

Reply:
Done.